website/docs/product/telemetry.md

```diff
@@ -8,58 +12,102 @@ Appsmith is a lightweight proxy and **does not capture** any data returned by yo
 
 ### Keep-alive ping
 
-The Appsmith server sends a keep-alive ping every 6 hours to indicate that it's still running without any errors & captures anonymous usage data around which plugins are being used.
+The Appsmith server sends a keep-alive ping every 6 hours to indicate that it's still running without any errors & captures anonymous usage data around which plugins are being used. This data is collected irrespective of whether telemetry is turned on or off.
```

Comment:
This ping is every 2 hours today, actually I had updated that in the Notion doc but missed checking it here. Also can we just say "The Appsmith server sends a keep-alive ping every 2 hours to check it instance is still running without any errors", we don't need any of the text written after that

Reply:
Will rewrite this.

Reply:
Though there is no checking here - the server is sending a ping to indicate it's alive.